.github/workflows/pg-9-migtests.yml

```diff
@@ -0,0 +1,110 @@
+name: "PG: Migration Tests"
```

Comment:
if we are going to have two workflows for PG.
Can we name it like:
PG[9]: Migration Tests
PG[13]: Migration Tests